self_balancing_tree.py

```diff
@@ -0,0 +1,128 @@
+# Hacker rank problem link: https://www.hackerrank.com/challenges/self-balancing-tree
+
+
+class Node:
+    def __init__(self,data , left=None, right=None):
+        self.data = data
+        self.height = 0
+
+
+def height(root):
+    if root:
+        return 1 + max(height(root.left), height(root.right))
+    return -1
+
+def updateHeight(root):

```

Comment:
This function is a misnomer as it doesn't actually update anything. Also it's a copy and paste of height above.